src/main/java/org/traccar/config/Keys.java

```diff
@@ -687,6 +687,36 @@ private Keys() {
             "sms.http.template",
             Collections.singletonList(KeyType.GLOBAL));
 
+    /**
+     * AWS Access Key with SNS permission.
+     */
+    public static final ConfigKey<String> AWS_ACCESS_KEY = new ConfigKey<>(
+            "aws.access",
+            Collections.singletonList(KeyType.GLOBAL));
+
+    /**
+     * AWS Secret Access Key with SNS permission.
+     */
+    public static final ConfigKey<String> AWS_SECRET_KEY = new ConfigKey<>(
+            "aws.secret",
+            Collections.singletonList(KeyType.GLOBAL));
+
+    /**
+     * SNS Enable switch.
+     * Use 'enable' to send message with SNS or 'disable' to stop using SNS
+     */
+    public static final ConfigKey<String> AWS_SNS_ENABLED = new ConfigKey<>(
+            "aws.sns.enabled",
```

Comment:
We are trying to reduce number of config parameters. Let's use presence of one of the other keys as "enabled" flag.

Reply:
I have gone through the config variables but I couldn't find one which represents the specific/meaningful description to enable/disable this feature.

Reply:
I meant to use other keys, like `region` or something.